applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -6,6 +6,12 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
 
     solver_type = solver_settings["solver_type"].GetString()
 
+    if solver_type == "ale_fluid":
+        KratosMultiphysics.CheckRegisteredApplications("MeshMovingApplication")
+        from KratosMultiphysics import MeshMovingApplication
+        import ale_fluid_solver
```

Comment:
Do you think it would make sense to put the ale_fluid_solver in FluidDynamicsApplication and restrict MeshMovingApplication to mesh motion? I'm asking because it seems like we have spread things around a bit between the two applications concerning ALE.

Reply:
fine for me, do you agree @jcotela ?

Reply:
this is also how it is done for the fm-ale embeddeds solver

I left the current ale-fluid solver in the MehMovingApp because of historical reasons

Reply:
I would be ok with either MeshMovingApplication->FluidDynamicsApplication or MeshMovingApplication<-FluidDynamicsApplication, but I would prefer to avoid MeshMovingApplication<->FluidDynamicsApplication for maintenance.

Reply:
Well I guess it made sense when the MeshMoving was called ALEApplication... I could be convinced to put ALE things in FluidDynamics, but we should try not to force the dependency.

Since many FluidDynamics things work without any ALE, users should be able to work normally without having to even compile the MeshMoving if they don't need those specific features.

Reply:
> Since many FluidDynamics things work without any ALE, users should be able to work normally without having to even compile the MeshMoving if they don't need those specific features.

Sure, otherwise we won't even pass travis :)

I will put it there then since also @rubenzorrilla does the same with the embedded stuff